projects/rocprofiler-sdk/source/lib/python/rocpd/libpyrocpd.cpp

```diff
@@ -469,12 +468,151 @@ PYBIND11_MODULE(libpyrocpd, pyrocpd)
                         auto regions = rocpd::sql_generator<rocpd::types::region>{
                             conn, select_guid_nid_pid("regions"), region_order_by};
 
+                        const std::string create_samples_view =
+                            "CREATE TEMP VIEW "
+                            "    `samples_temp` AS "
```

Comment:
```suggestion
                            "    `samples_schema_3_0` AS "
```
Use a better name instead of "samples_temp".  Since the next schema will be version bumped, we can refer to the 3.0 schema for these.

Reply:
Done